tools/beerocks_analyzer/beerocks_analyzer_widget.py

```diff
@@ -184,19 +195,25 @@ def getCommandLineArgs(self, argv):
                     self.logger.error("Error, unknown argument ({:d}) --> {}".format(i, arg_i))
                     self.printUsage()
                     sys.exit(0)
-            except Exception as e: # TODO: too broad exception
+            except Exception as e:  # TODO: too broad exception
                 self.logger.error("Error(2), in argument ({:d}) --> {}".format(i, arg_i))
                 self.printUsage()
                 sys.exit(0)
 
     def printUsage(self):
         self.logger.info("usage: BeeRocksAnalyzer [options]\n")
-        self.logger.info("    -f=<log file>                                         - input log file\n")
-        self.logger.info("    -plot<fig num><sub plot>=<Label 1>,<Label 1>,..       - enable and configure plots\n")
-        self.logger.info("    -realtime=<update interval sec>,<samples>             - enable real time plot\n")
-        self.logger.info("    -print_vals                                           - print log values to console\n")
-        self.logger.info("    -print_labels                                         - print log labels to console\n")
-        self.logger.info("    -output_csv                                           - output params to csv file\n")
+        self.logger.info(
+            "    -f=<log file>                                     - input log file\n")
```

Comment:
Same here, if you have the time/opportunity, please remove all the extra spaces after the single dash.